docs/docs/Components/components-tools.md

```diff
@@ -257,25 +259,56 @@ This component allows you to call the Serper.dev Google Search API.
 | results | List[Data]| List of search results               |
 | tool    | Tool      | Google Serper search tool for use in LangChain|
 
-## MCP Tools (stdio)
 
-This component connects to a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) server over `stdio` and exposes its tools as Langflow tools to be used by an Agent component.
+## MCP Server
```

Comment:
You capitalized "Server" here, but not in the body text. I don't think it should be capitalized here.